### PR TITLE
Made more unit test files to use the assert macro

### PR DESCRIPTION
**Pull request by Unique-Usman**
@bluca @DaanDeMeyer, I was thinking it will be okay to make more test files used the macro few days ago. I had some chance after class today so, I started with it. I hope it is okay to have that.

### CODE REVIEW

**Review by DaanDeMeyer — APPROVED** [review]

File: pkg/debian
Comment (YHNdnzj) [inline pkg/debian:1]: Hmm? Why is this changed?

**Comment by YHNdnzj** [comment]
Please give it a better commit message next time.

**Comment by Unique-Usman** [comment]
> Please give it a better commit message next time.

Noted, thank you very much.

**Comment by Unique-Usman** [comment]
> @Unique-Usman Thank you! However, could you leave the rest for others to do? This is a good beginner task so I'd like to keep the remaining ones for other beginners to do to get acquainted with systemd development.

Noted, this is exactly right. Thank you.

File: pkg/debian
Comment (Unique-Usman) [reply pkg/debian:1]: Ohh, I did not intend to add this at all. @DaanDeMey already opened a PR to make the changes. Thank you.

File: src/test/test-af-list.c [hunk 1]
```diff
@@ -16,14 +16,14 @@ static const struct af_name* lookup_af(register const char *str, register GPERF_
 TEST(af_list) {
         for (unsigned i = 0; i < ELEMENTSOF(af_names); i++) {
                 if (af_names[i]) {
-                        assert_se(streq(af_to_name(i), af_names[i]));
+                        ASSERT_STREQ(af_to_name(i), af_names[i]);
                         assert_se(af_from_name(af_names[i]) == (int) i);
```
Comment (mrc0mmand) [inline src/test/test-af-list.c:20]: Hmm, why is this not converted? Similarly to other files - only some of the asserts seem to be converted to the new macros in each file.

Comment (daandemeyer) [reply src/test/test-af-list.c:20]: I actually don't want us to convert all of them. This is a great first task for new contributors, so I figure let's not insist on converting all of them.

Comment (Unique-Usman) [reply src/test/test-af-list.c:20]: The current available macros do not cover for all instances also. A great first task for new contributors as @DaanDeMeyer made mentioned.

Comment (mrc0mmand) [reply src/test/test-af-list.c:20]: I understand that, but maybe it would be great to do the migration on file-to-file basis, since now the tests use both variants which is a bit messy.

Comment (Unique-Usman) [reply src/test/test-af-list.c:20]: I had the intention of converting all the unit test file, the changes is basically for the first 20 files in the src/test directory.
